docs/Coding-Standards/storybook.stories.mdx

```diff
@@ -0,0 +1,415 @@
+
+## Storybook Tutorial
+
+### Introduction - Essentials
+
+Storybook works with path glob patterns that point to stories files. The glob patterns can be found in `.storybook/main.js`:
+
+```jsx
+stories: [
+    '../docs/**/*.stories.mdx',
+    '../src/app/legacy/components/**/*.stories.@(t|j)sx',
+    '../src/app/legacy/containers/**/*.stories.@(t|j)sx',
+    '../src/app/components/**/*.stories.@(t|j)sx',
+    '../src/app/pages/**/*.stories.@(t|j)sx',
+    './DocsDecorator/**/*.stories.@(t|j)sx',
+    './SidebarLabel/**/*.stories.@(t|j)sx',
+],
+```
+
+For any of our stories or documentation to correctly display in storybook we must ensure that the folder where they are stored is pointed by any of the above path glob patterns.
+
+### How to add documentation
+
+Make sure that the place where you store your documentation is pointed by any of the glob patterns in `.storybook/main.js`.
+Then create an `.stories.mdx` file and write your documentation in normal markdown. Once you have finished, make sure to add these lines at the top of the file:
+
+```jsx
+import { Meta } from '@storybook/addon-docs';
+
+//Title follows the path structure where the documentation needs to be stored. More about this is explained in later sections
+<Meta title="Coding Standards/Storybook" />;
+
+Your documentation should follow markdown format, documentation can be found [here](https://www.markdownguide.org/basic-syntax/)
+```
+
+The above code will allow storybook to grab the markdown you wrote below the `Meta` tag and display it in a story documentation.
+
+### Creating a Basic component's story
+
+To create a story for one of your components, co-locate a file called `index.stories.tsx` where your component's entry point (index.tsx/jsx) is located.
+Inside this folder import your component and do a default export of the object showed below:
+
+```jsx
+import MyComponent from '.';
+
+export default {
+  // Title follows the path structure where the story needs to be stored. More about this is explained in later sections
+  title: 'myComponent',
+  // the component you want to display.
+  component: MyComponent,
+};
+```
+
+### Creating story for a component with variants
+
+If your component has different behaviours depending on the props passed to it, you can create more variants with different props passed to it. All these variants will be displayed in the component's story.
+
+```jsx
+// Same as above but we will need React
+import React from 'react';
+import MyComponent from '.';
+
+export default {
+  title: 'myComponent',
+  component: MyComponent,
+};
+
+// Export an arrow function with the variants of your components
+export const Variant1 = () => <MyComponent prop1={variant1} />;
+
+// The name declaration of this function will also be the name that storybook will give to this variant
+export const Variant2 = () => <MyComponent prop1={variant2} />;
+```
+
+### Creating story for a component that require providers
```

Comment:
Unrelated to this ticket, could we use the [global decorator](https://storybook.js.org/docs/react/writing-stories/decorators#global-decorators) to handle all of our providers, so we don't have to include them individually for each story that needs them? Similar to our react-testing-libray-with-providers [setup](https://github.com/bbc/simorgh/blob/latest/src/app/components/react-testing-library-with-providers.tsx)

Reply:
Alright, a global providers proposal for storybook has just been pushed. Let me know what you think about it.